src/hb-config.hh

```diff
@@ -60,6 +61,7 @@
 #define HB_NO_GETENV
 #define HB_NO_LAYOUT_UNUSED
 #define HB_NO_MATH
+#define HB_NO_MMAP
```

Comment:
Or having both in lean? My reasoning was it can work without mmap so it is lean but disabling the whole API in tiny

Reply:
Right. TINY itself doesn't do these; LEAN does.  IMplement it by disabling the feature now.  Then we switch all to remove the API.  I don't think we need both.